src/Analyzer/ClassDescription.php

```diff
@@ -141,4 +146,18 @@ public function isAbstract(): bool
     {
         return $this->abstract;
     }
+
+    public function getAnnotation(): string
```

Comment:
this is a container of multiple annotations? maybe this should be plural to be more coherent with hasAnnotation?

Reply:
yes, I agree, I fix it immediately.